packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -701,7 +701,14 @@ class DropdownButton<T> extends StatefulWidget {
     this.focusColor,
     this.focusNode,
     this.autofocus = false,
-  }) : assert(items == null || items.isEmpty || value == null || items.where((DropdownMenuItem<T> item) => item.value == value).length == 1),
+  }) : assert(items == null || items.isEmpty || value == null ||
+              items.where((DropdownMenuItem<T> item) {
+                return item.value == value;
+              }).length == 1,
+                'There should be one item with [DropdownButton]\'s value: $value. \n'
+                'There might be no [DropdownButtonItem]s with this value, or '
```

Comment:
This is stated in the same way that the expected (correct) configuration is stated in the first sentence. Might be better to say something like: "Either zero or 2 or more [DropdownMenuItem]s ..."